docs/create_user.md

```diff
@@ -0,0 +1,9 @@
+# Creating new users
+
+If you want to allow a new user to access your web app, follow these steps:
+
+1. cd to root of Omnilog
```

Comment:
Could add a step here that you'll have to run `yarn` at the project root for dependancy installation first?

Reply:
Added a commit!